ofp4/src/main.rs

```diff
@@ -599,18 +702,35 @@ fn main() -> Result<()> {
         .unwrap();
     server.start();
 
+    if p4_port == 0 {
```

Comment:
from here I would move the code to a separate function.

Reply:
Done, I pushed a commit for this.